scripts/sanitycheck

```diff
@@ -1159,8 +1159,11 @@ class MakeGenerator:
 
             cmd = ["make", "-k", "-j",
                    str(int(CPU_COUNTS * loadmul)), "-f", tf.name, "all"]
+            # assure language neutral == English environemnt
+            make_env = os.environ.copy()
+            make_env['LANG'] = 'en'
```

Comment:
Setting `LC_MESSAGES` to `C` (the standard POSIX locale, which probably means English) might be better.

`LANG` takes the name of a locale. Can run `locale -a` to see the ones available on your system.

If `en` works, it's probably because the locale isn't recognized, making it fall back to the `C` locale.

See http://pubs.opengroup.org/onlinepubs/7908799/xbd/envvar.html.